config/locales/en.yml

```
@@ -3440,7 +3440,7 @@ See the %{link} to find out more about %{sitename}'s features and to start using
           processing: "processing"
           void: "void"
           invalid: "invalid"
-        quantity_adjusted: "Insufficient stock available. Line item updated to maximum available quantity."
+        quantity_unavailable: "Insufficient stock available. Line item unsaved!"
```

The unsaved value has been updated to maximum available, so it would be nice to say so. I'd suggest something like this, which also gives the user direction on what to do next:
```suggestion
        quantity_unavailable: "Insufficient stock available, only %{count} left. Please try again."
```

Of course, that will require some more code changes. I'm not sure if that will be relevant in future changes so will let you consider if it's worth doing or not.

---

Very good idea but I don't want to invest more time into this legacy code. People will see the max quantity in the new unsaved value.